### PR TITLE
fix: Added missing help command

### LINKED ISSUE #1826: Add -h help flag to all commands
**Author the user story for this feature**

As a user, I want to see help on all commands so I can learn how to use Flank properly.

**Describe the solution**

The `-h` help flag is only implemented on the run commands. For example:
` java -jar flank.jar ios run -h`

All commands should have support for the `-h` help command. Currently passing `-h` generates an error on most Flank commands.

```
$ java -jar flank.jar -h
Unknown option: '-h'
```

```
$ java -jar flank.jar ios -h
Unknown option: '-h'
```

**Describe alternatives considered**

Not having help support on all the commands.

### PR DESCRIPTION
**Pull request by adamfilipow92**
Fixes #1826 

1. All tests pass
1. All commands displays help

## Checklist

- [X] Unit tested

### CODE REVIEW

File: test_runner/src/main/kotlin/ftl/util/PrintHelp.kt [hunk 1]
```diff
@@ -2,8 +2,14 @@ package ftl.util
 
 import picocli.CommandLine
 
-interface PrintHelp : Runnable {
+abstract class PrintHelp : Runnable {
```
Comment (piotradamczyk5) [inline test_runner/src/main/kotlin/ftl/util/PrintHelp.kt:5]: How about `PrintHelpCommand`?

File: test_runner/src/test/kotlin/ftl/cli/CliTest.kt [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    private fun Collection<CommandLine>.getAllSubCommands(): Collection<CommandLine> =
+        if (any()) (this + flatMap { it.subcommands.values }.getAllSubCommands())
+        else this
+
+    private fun Collection<CommandLine>.getCommandWithoutHelp() = map { command ->
+        command to command.commandSpec.args().mapNotNull { arg -> arg as? CommandLine.Model.OptionSpec }
+            .any { arg -> arg.names().contains("-h") || arg.names().contains("--help") }
+    }.filter { !it.second }.map { it.first }
```
Comment (piotradamczyk5) [inline test_runner/src/test/kotlin/ftl/cli/CliTest.kt:32]: could we name these `first` and `second`?

**Review by piotradamczyk5 — APPROVED** [review]

**Review by Sloox — COMMENTED** [review]
@adamfilipow92 Does each command actually have a help description or do they all print out "Prints this help message"?

**Comment by adamfilipow92** [comment]
> @adamfilipow92 Does each command actually have a help description or do they all print out "Prints this help message"?

All commands have a help description. Here is an example: 

```
flank.jar
 [-hv] [--debug] [COMMAND]
      --debug     Enables debug logging
  -h, --help      Prints this help message
  -v, --version   Prints the version
Commands:
  firebase
  corellium
  ios
  android
  refresh            Downloads results for the last Firebase Test Lab run
  cancel             Cancels the last Firebase Test Lab run
  auth               Manage oauth2 credentials for Google Cloud
  provided-software
  network-profiles   Explore network profiles available for testing.
  ip-blocks          Explore IP blocks used by Firebase Test Lab devices.

Total run duration: 0m  0s

```

As you see ```Prints this help message``` describes help parameters.

**Review by Sloox — APPROVED** [review]
👍

**Pull request merged**